sample_code/ep148/README.md

```diff
@@ -0,0 +1,71 @@
+# [customize ANY website (beginner - intermediate)](https://youtu.be/Cmz6YGzF0PM)
+
+Today I show how to hack on any behaviour you want to a website! user scripts are one of my favorite productivity boost tools.  I show a few ways to use greasemonkey/tampermonkey to improve some websites and hopefully give some inspiration for things you can do too!
+
+## Interactive examples
+
+### UserScript
+
+Refresh:
+
+```js
+// ==UserScript==
+// @name         refresh
+// @namespace    http://tampermonkey.net/
+// @version      0.1
+// @description  try to take over the world!
+// @author       You
+// @match        https://old.reddit.com/r/friendsafari/new/
+// @grant        none
+// ==/UserScript==
+
+(function() {
+    'use strict';
+
+    window.setTimeout(() => window.location.reload(), 5 * 1000);
+})();
+```
+
+Pokemon:
+
+```js
+// ==UserScript==
+// @name         highlight pokemon
+// @namespace    http://tampermonkey.net/
+// @version      0.1
+// @description  try to take over the world!
+// @author       You
+// @match        https://old.reddit.com/r/friendsafari/new/
```

Comment:
This will world only for the main page of the subreddit.
I guess that it can be a little confusing for someone that doesn't know that you need wildcard at the end of the url.
For example `https://old.reddit.com/r/friendsafari/new/*`

Reply:
yeah my actual script for this uses `/r/friendsafari/*`